queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE NAME LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered=true AND escape_attempt<3;
+SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
+SELECT name,escape_attempt FROM animals WHERE weight_kg>10.5;
+SELECT * FROM animals WHERE neutered=true;
+SELECT * FROM animals WHERE name<>'Gabumon';
+SELECT * FROM animals WHERE weight_kg>=10.4 AND weight_kg<=17.3;
```

Comment:
- [OPTIONAL] Although date of birth is not required in this query.  but because you are showing it, kindly consider updating the screenshot of this last query because the screenshot is showing the wrong outdated date of birth of Pikachu.

Reply:
@Zuhaib042 , 
thanks so much. I will do that